### PR TITLE
Remove config_opts and config file

### PR DESCRIPTION
**Pull request by mwaskom**
This addresses #120

Values that were previously selected using a ``config_opts`` dictionary in the ``Brain`` constructor are now keyword arguments. This should make it easier to use tab-completion in IPython, and will generally simplify your scripts. To ease transition, a ``config_opts`` argument will still be caught, and its entries will be used (overriding the keyword arguments), but it will be removed in a future release.

The ability to set default values in a config file has been removed. While convenient, this approach encourages code that is not fully reproducible. While existing code that was written expecting a config file will still run, the visualization will fall back to default values. These should be updated directly in your plotting scripts.

While not directly related to these changes, this  PR also adds a new feature: the ``cortex`` keyword argument can now be a mayavi colormap name or a ``(colormap, min, max, reverse)`` tuple for full control.

### CODE REVIEW

**Comment by mwaskom** [comment]
I think I've got everything and the tests pass on my machine, but please take a look.

File: doc/documentation/custom_viz.rst [hunk 1]
```diff
@@ -12,7 +12,17 @@ have to delve into the underlying engine to get a different look.
...
+The display background can take any valid matplotlib color (i.e.,
+it can be a tuple of rgb values, an rgb hex string, or a named HTML
+color.
+
+Changing the display size
+-------------------------
+
+The default display window is 800px by 800px, but this is also
+customizable. You can set the size of a square window with the ``size``
+parameter or make a rectangular window with the ``width`` and ``height``
```
Comment (Eric89GXL) [inline doc/documentation/custom_viz.rst]: While we're at it, how about we deprecate `size`, too? Just about every other viz package uses `width` and `height`, and it's more pythonic to have one way of doing it.

**Comment by Eric89GXL** [comment]
Other than my one idea, LGTM +1 for merge

File: doc/documentation/custom_viz.rst (hunk 1 above)
Comment (mwaskom) [inline doc/documentation/custom_viz.rst]: I agree that in general it feels like a clunky API, and that just taking width and height is more common (although usually that's `size = (width, height)`, I feel like).

Comment (Eric89GXL) [inline doc/documentation/custom_viz.rst]: Ahh, right. So maybe deprecate `width` and `height` and allow `size` to be an int or tuple/list of two int?

Comment (mwaskom) [inline doc/documentation/custom_viz.rst]: I think this is probably the best API, although I think it produces the ugliest code for handling backwards compatibility properly.

Comment (agramfort) [inline doc/documentation/custom_viz.rst]: +1 for size with tuple as figsize in matplotlib

**Comment by agramfort** [comment]
besides diff looks good

**Comment by mwaskom** [comment]
OK I think that will do the trick.

**Comment by mwaskom** [comment]
damn @coveralls u strict

**Comment by Eric89GXL** [comment]
We can opt to ignore it, though :) Good to go from your end, then?

**Comment by Eric89GXL** [comment]
Oh right "yes" because of previous comment, merging. Thanks for cleaning this up.

**Comment by agramfort** [comment]
Yes

**Pull request merged**